base_invader/controllers/main.py

```diff
@@ -66,19 +66,19 @@ def _process_method(self, service_name, method_name, _id=None,
             return self.make_response(res)
 
     @route([
-        ROOT_PATH + '<string:_service_name>/',
+        ROOT_PATH + '<string:_service_name>',
```

Comment:
Here I would keep the '/' so it works with and without / (but that's nitpicking, I admit).

Reply:
Whether or not the route ends with or without '/' the service's method can be reached with 'ping/' or 'ping'